relayer/src/utils/createNonce.ts

```diff
@@ -0,0 +1,3 @@
+import nacl from "tweetnacl";
+export const createNonce = (): string =>
```

Comment:
why is this in it's own file?
(I believe this is used in one place. Thus, let's put it there.)

Reply:
done